manipulation/packages/arm_pkg/scripts/grasp_markers.py

```diff
@@ -0,0 +1,252 @@
+#!/usr/bin/env python3
+import rclpy
+from rclpy.node import Node
+from visualization_msgs.msg import Marker, MarkerArray
+from sensor_msgs.msg import PointCloud2
+from frida_interfaces.srv import GraspDetection, ReadPcdFile
+import colorsys
+import tf_transformations
+import numpy as np
+
+
+class GraspVisualizer(Node):
```

Comment:
Question here @JLDominguezM is this node going to be a service? or is it intended to be used in other way???